e2e/migration_common.go

```diff
@@ -0,0 +1,20 @@
+package e2e
+
+import (
+	machinev1beta1 "github.com/openshift/api/machine/v1beta1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+)
+
+const (
+	// SynchronizedCondition indicates that a machine resource has been successfully synchronized between MAPI and CAPI during migration..
+	SynchronizedCondition machinev1beta1.ConditionType = "Synchronized"
```

Comment:
Do we not export these from the MAPI libraries? Can we import them like we do with the CAPI one below? 


If not, please ignore!

Reply:
yes, we cant export these like CAPI one.